### ISSUE TITLE
Error in Analysis API

### ISSUE THREAD

**Issue by InsanesTheName** [tenable/pyTenable#56]
**Describe the bug**
Calling `sc.analysis.scan(scan_id)` throws the following error:
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/lib/python3.5/site-packages/tenable/sc/analysis.py", line 490, in scan
    return self.vuln(*filters, **kw)
AttributeError: 'AnalysisAPI' object has no attribute 'vuln'
```
Looks like perhaps it should be `vulns()`?

**To reproduce**
1. Create a TenableSC object and authenticate with `login()`
1. Call `analysis.scan(scan_id)`
1. See error

**Expected behavior**
Vulnerability data from the provided `scan_id` is returned

**System information (please complete the following information):**
 - OS: Ubuntu 16.04
 - Architecture: 64bit
 - Version: pyTenable 0.3.6

**Comment by SteveMcGrath** [comment]
feel free to pull from the github source `pip install git+git://github.com/tenable/pytenable.git#egg=pytenable
` as this should have addressed it.

**Comment by SteveMcGrath** [comment]
This was released in 0.3.8.  closing
